examples/README.md

```diff
@@ -14,14 +14,14 @@ Each examples is designed to be self-contained and easily forkable, while reprod
 | Example | Description | Features |
 | ------- | ----------- | -------- |
 | [cifar10](examples/cifar10/README.md) | Wideresnet & pyramidnet implementations with shake-shake / shake-drop regularization for CIFAR-10 classification | Single host SPMD, tfds `cifar10`, custom preprocessing |
```

Comment:
I think you missed this?

Reply:
I did.. - my bad. The updated commit should now fix this as well.